addons/apecloud-mysql/scripts/etcd-clean.sh

```diff
@@ -18,11 +23,17 @@ servers=${servers:1}
 echo $servers
 
 echo "Deleting all keys with prefix /vitess/${KB_NAMESPACE}/${KB_CLUSTER_NAME} from Etcd server at ${endpoints}..."
-etcdctl --endpoints $servers del /vitess/${KB_NAMESPACE}/${KB_CLUSTER_NAME} --prefix
+
+# Set different deletion methods according to different etcdctl versions.
+if [[ ${ETCDCTL_API} == "3" ]]; then
```

Comment:
it's better changed to :

```
if [[ ${ETCDCTL_API} == "2" ]]; then
    ...
else
   ...
fi
```

so that if ETCDCTL_API is not set in extra_env, "3" is the default value.

Reply:
yes I will modify it